src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +121,27 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        //
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {

```

Comment:
@Vidya2606 can you add a unit test case for this method, since it is public method, it should be easily testable. Thank you for the PR.